.travis.yml

```diff
@@ -18,87 +15,56 @@ env:
     - COVERALLS_ENABLED="false"
 
 matrix:
-
   fast_finish: true
-
   include:
-
-    - php: 7.1
     - php: 7.1
       env:
-        - PHPUNIT_ENABLED="false"
         - PHPCSFIXER_ENABLED="true"
-    - php: 7.1
-      env:
-        - SYMFONY_VERSION=4.0.*
-        - COMPOSER_FLAGS="--prefer-lowest"
-    - php: 7.2
-    - php: 7.2
-      env: SYMFONY_VERSION=3.4.*
-    - php: 7.2
-      env: SYMFONY_VERSION=4.0.*
```

Comment:
what is the reasoning for not testing each allowed symfony minor version? that can sometimes find issues with symfony version differences.

Reply:
Doesn’t the Symfony BC promise enough? If not what would be the rules? I often see hard to maintain Travis matrices when testing against each Symfony version.

What about testing each supported and maintained Symfony version with the lowest (or highest) supported php version?

Reply:
Also, what about dropping Sf 4.0, 4.1 and 4.2 as those are no longer maintained? 4.2 could be kept as it’s still maintained for security.

This would help to keep the matrix clean.

Reply:
i have seen things go wrong in unexpected ways before, would prefer to test each minor version we claim to support. +1 on dropping 4.0 and 4.1 support. 4.2 sounds like we should keep it while it still gets security support.